### PR TITLE
readme: add `props`

### PR DESCRIPTION
**Pull request by StunxFS**
Property support like Java for V

### CODE REVIEW

**Comment by Delta456** [comment]
Can you add more detail about `Property support` in `Java` as it will be helpful for others who ain't using Java to know what it is and may find it useful in future?

**Comment by StunxFS** [comment]
Ready.

File: README.md [hunk 1]
```diff
@@ -89,6 +89,7 @@
 - [vast](https://github.com/lydiandy/vast) - A simple tool for vlang, generate v source file to AST json file.
 - [symlinker](https://github.com/serkonda7/symlinker) - A small Linux tool to manage symlinks.
 - [runner](https://github.com/Naheel-Azawy/runner) - A tool that automates running/compiling code written in various programming languages.
+- [props](https://github.com/StunxFS/props) - Property support like Java for V
```
Comment (Delta456) [inline README.md:92]: Please add a `.` at the end of the sentence.

Comment (StunxFS) [reply README.md]: Ready

**Comment by alexesprit** [comment]
You used the wrong section. `props` is not an application, but a library, and should be placed in the "Libraries" section.

**Comment by StunxFS** [comment]
True! I missed the detail, thank you very much. I'll correct it.

**Pull request merged**